gulpfile.js

```diff
@@ -141,9 +130,9 @@ gulp.task('package:prepare', () => {
     .pipe(gulp.dest(paths.package + 'public/sass/'))
 })
 
-gulp.task('package', cb => {
-  runsequence('package:prepare', cb)
-})
+gulp.task('package', gulp.series(
```

Comment:
I don't think this is something that needs fixing in this PR, but because this task doesn't delete the contents of the `package/public/sass` directory, files don't get deleted from the package when they're deleted from `/assets/sass` – so  [`_shame.scss`](https://github.com/alphagov/govuk_elements/blob/77783b1dfecb74048f8824a720aa8da9d9cdbba5/packages/govuk-elements-sass/public/sass/elements/_shame.scss) is still in the package folder, despite having been deleted in #611.

Reply:
I've had a go at resolving this in https://github.com/alphagov/govuk_elements/pull/635/commits/93ffb917ee66f52c4e0f516c48a18cb5806119dd. But I could split it into a separate PR? (If we keep it here I'll remove https://github.com/alphagov/govuk_elements/pull/635/commits/9993eee8af6eb9d60228ac828e63ec4aea016620)

Reply:
Yep, let's split this out and worry about it later – I don't think that's quite right as a lot of `/package` is not generated from anywhere (e.g. `/package/package.json`)

Reply:
Yes that's the reason only `packages/govuk-elements-sass/public` is removed and the files at `/packages/` level aren't touched. I'll raise the PR once this is merged.

Reply:
Ah, sorry, I think I was thrown by the commit message. Let's review it separately 👍

Reply:
Yeah the commit message could have been clearer. Cool I'll raise that PR separately.